unattended_scripts/install_functions/opendistro/common.sh

```diff
@@ -114,6 +118,9 @@ restoreWazuhrepo() {
         eval "sed -i 's/-dev//g' ${file} ${debug}"
         eval "sed -i 's/pre-release/4.x/g' ${file} ${debug}"
         logger "Done"
+        ((progressbar_status++))
+    else 
+        ((progressbar_total--))
```

Comment:
Remove else, and just increment the progress_bar if development task is done

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1080/commits/b6529a2b3b0eeb287bd573b77284f34b2fef99a6